optimum/intel/neural_compressor/configuration.py

```diff
@@ -90,3 +93,201 @@ def _create_distillation_config(config: Union[Dict, DistillationConfig]):
+    Args:
+        bits (`int`, defaults to 4):
+            The number of bits to quantize to.
+        sym (`bool`, defaults to `False`):
+            Whether to use symmetric quantization on the weights.
+        num_samples (`int`, *optional*):
+            The maximum number of samples composing the calibration dataset.
+        batch_size (`int`, *optional*):
+            The batch size number of calibration dataset.
+        seq_len (`int`, *optional*):
+            The sequence length number of calibration dataset.
+        tokenizer (`str`, *optional*):
+            The tokenizer used to process the dataset. You can pass either:
+                - A string, the *model id* of a predefined tokenizer hosted inside a model repo on huggingface.co.
+                    Valid model ids can be located at the root-level, like `bert-base-uncased`, or namespaced under a
+                    user or organization name, like `dbmdz/bert-base-german-cased`.
+                - A path to a *directory* containing vocabulary files required by the tokenizer, for instance saved
+                    using the [`~PreTrainedTokenizer.save_pretrained`] method, e.g., `./my_model_directory/`.
+        dataset (`str or List[str]`, *optional*):
+            The dataset used for weight-only quantization with neural-compressor, the default is "NeelNanda/pile-10k".
+        group_size (`int`, *optional*):
+            The group size to use for quantization. Recommended value is 128 and -1 uses per-column quantization.
+        use_layer_wise(`bool`, *optional*):
+            Enable quantize model per layer. `model_name_or_path` parameter is necessary when used `use_layer_wise`, it indicates the model
+            local folder name or the name on remote model hub.
+        quant_method (`str or INCQuantizationMethod`, defaults of INCQuantizationMethod.RTN):
+            Weight compression method to apply. Possible options:
+                - "rtn": weight quantization based round to nearest will be applied.
+                - "gptq": A new one-shot weight quantization method based on approximate second-order information,
+                    that is both highly-accurate and highly efficient. The weights of each column are updated based on
+                    the fixed-scale pseudo-quantization error and the inverse of the Hessian matrix calculated from the activations.
+                    The updated columns sharing the same scale may generate a new max/min value,
+                    so the scale needs to be saved for restoration.
+                - "autoround": AutoRound is an advanced weight-only quantization algorithm for low-bits LLM inference.
+                    It's tailored for a wide range of models and consistently delivers noticeable improvements,
+                    often significantly outperforming SignRound with the cost of more tuning time for quantization.
+    """
+
+    def __init__(
+        self,
+        bits: int = 4,
```

Comment:
could make sense to have 8 as default
```suggestion
        bits: int = 8,
```